src/pages/Pool/PositionPage.tsx

```diff
@@ -178,7 +179,7 @@ function CurrentPriceCard({
           <Trans>Current price</Trans>
         </ExtentsText>
         <ThemedText.DeprecatedMediumHeader textAlign="center">
-          {(inverted ? pool.token1Price : pool.token0Price).toSignificant(6)}{' '}
+          {formatPrice(inverted ? pool.token1Price : pool.token0Price, NumberType.TokenTx)}{' '}
```

Comment:
I bet using `formatTickPrice` would be a better experience here. The current implementation doesn't use `,` to separate thousands.